specification/monitor/data-plane/preview/2018-09-01-preview.json

```diff
@@ -0,0 +1,1927 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure Metrics",
+    "version": "2018-09-01-preview",
+    "description": "A client for issuing REST requests to the Azure metrics service.",
+    "x-ms-code-generation-settings": {
+      "name": "AzureMetricsClient"
+    }
+  },
+  "host": "monitoring.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json; odata=minimalmetadata"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/{resourceProvider}/{resourceTypeName}/{resourceName}/metrics": {
+      "post": {
+        "summary": "Ingestion API used by Azure custom metrics\r\n            Responsible of below operation\r\n            1. Authorize the request\r\n            2. Deserialize payload request body\r\n            3. Perform validation of the payload\r\n            4. Commits the payload for metrics ingestion",
+        "operationId": "postSubscriptionsBySubscriptionIdResourcegroupsByResourceGroupNameProvidersByResourceProviderByResourceTypeNameByResourceNameMetrics",
```

Comment:
The ```operationId``` is used to generate method names. Please shorten it to something more descriptive.